cmd/cli/install.go

```diff
@@ -205,6 +207,7 @@ func (i *installCmd) run(config *helm.Configuration) error {
 		return err
 	}
 
+	log.Trace().Msgf("run(): start resolveValues()")
```

Comment:
@allenlsy would you be so kind to use logger package in this repo - initialized w/ the right string it provides a bit more context around these log lines:
https://github.com/openservicemesh/osm/blob/707593c3a291ec9ac4a3e92836d8776cb3d84ce3/pkg/envoy/rds/types.go#L4-L9

Reply:
Shashank mentioned [here](https://github.com/openservicemesh/osm/pull/3198#discussion_r624264762) that we have issue #912 for similar purpose. Will work on it separately.